examples/advanced/tidb-cluster.yaml

```diff
@@ -1,295 +1,872 @@
+    #   storage: 1Gi
+    # limits:
+    #   cpu: 2000m
+    #   memory: 2Gi
+
+    ## defines Kubernetes service for pd-server
+    ## Ref: https://kubernetes.io/docs/concepts/services-networking/service/
+    # service:
+    #   type: ClusterIP
+    #   annotations:
+    #     foo: bar
+    #   portName: client
+
+    #############################
+    # Advanced PD Configuration #
+    #############################
+
+    ## The following block overwrites TiDB cluster-level configurations in `spec`
+    # version: "v4.0.8"
+    # imagePullPolicy: IfNotPresent
+    # imagePullSecrets: secretName
+    # hostNetwork: false
+    # serviceAccount: advanced-tidb-pd
+    # priorityClassName: system-cluster-critical
+    # schedulerName: tidb-scheduler
+    # nodeSelector:
+    #   app.kubernetes.io/component: pd
+    # annotations:
+    #   node.kubernetes.io/instance-type: some-vm-type
+    # tolerations:
+    #   - effect: NoSchedule
+    #     key: dedicated
+    #     operator: Equal
+    #     value: pd
+    # configUpdateStrategy: RollingUpdate
+    # statefulSetUpdateStrategy: RollingUpdate
+
+    ## List of environment variables to set in the container
+    ## Note that the following env names cannot be used and will be overwritten by TiDB Operator builtin envs
+    ##   - NAMESPACE
+    ##   - TZ
+    ##   - SERVICE_NAME
+    ##   - PEER_SERVICE_NAME
+    ##   - HEADLESS_SERVICE_NAME
+    ##   - SET_NAME
+    ##   - HOSTNAME
+    ##   - CLUSTER_NAME
+    ##   - POD_NAME
+    ##   - BINLOG_ENABLED
+    ##   - SLOW_LOG_FILE
+    ## Ref: https://kubernetes.io/docs/tasks/inject-data-application/environment-variable-expose-pod-information/
+    # env:
+    # - name: MY_ENV_1
+    #   value: value1
+    # - name: MY_ENV_2
+    #   valueFrom:
+    #     fieldRef:
+    #       fieldPath: status.myEnv2
+
+    ## Custom sidecar containers can be injected into the PD pods,
+    ## which can act as a logging/tracing agent or for any other use case
+    # additionalContainers:
+    # - name: myCustomContainer
+    #   image: ubuntu
+
+    ## custom additional volumes in PD pods
+    # additionalVolumes:
+    # # this must match `name` in `additionalVolumeMounts`
+    # - name: nfs
+    #   nfs:
+    #     server: 192.168.0.2
+    #     path: /nfs
+
+    ## custom additional volume mounts in PD pods
+    # additionalVolumeMounts:
+    # # this must match `name` in `additionalVolumes`
+    # - name: nfs
+    #   mountPath: /nfs
+
+    ## Optional duration in seconds the pod needs to terminate gracefully. May be decreased in delete request.
+    ## Ref: https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/#hook-handler-execution
+    # terminationGracePeriodSeconds: 30s
+
+    ## PodSecurityContext holds pod-level security attributes and common container settings.
+    ## Ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
+    # podSecurityContext:
+    #   sysctls:
+    #   - name: net.ipv4.tcp_syncookies
```

Comment:
Do not use this example, use the example here https://github.com/pingcap/tidb-operator/blob/master/examples/aws/tidb-cluster.yaml#L65.
`net.ipv4.tcp_syncookies` is not namespace scoped for kernel lower than 4.4.

Reply:
```
    #   - name: net.core.somaxconn
    #     value: "32768"
```
Use this?